src/arrayMethodSort.js

```diff
@@ -4,8 +4,32 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
Using `__proto__` to add methods to prototypes is discouraged in modern JavaScript. It is better to use `Object.defineProperty(Array.prototype, 'sort2', {...})` to define the method more safely and avoid potential conflicts.